src/components/Item/Item.component.jsx

```diff
@@ -68,47 +100,80 @@ const Item = props => {
       });
   };
 
-  useEffect(() => {
-    if (over24 === false) {
-      document.getElementById(itemId).style.textDecoration = 'line-through';
-    }
-    // To highlight the items based on resupplyPeriod
-    if (resupplyPeriod === 7) {
-      document.getElementById(itemId).setAttribute('class', 'list__item--soon');
-      return;
-    } else if (resupplyPeriod === 14) {
-      document
-        .getElementById(itemId)
-        .setAttribute('class', 'list__item--kind-of-soon');
-      return;
-    } else if (resupplyPeriod === 30) {
-      document
-        .getElementById(itemId)
-        .setAttribute('class', 'list__item--not-soon');
-      return;
-    } else if (
-      numberOfPurchases < 2 ||
-      lastPurchasedTimeInSeconds > 2 * nextPurchaseInterval
-    ) {
-      document
-        .getElementById(itemId)
-        .setAttribute('class', 'list__item--inactive');
-    }
-  });
+  // Variable set up for easy theming whenever we want to get rid of these LOVELY colours LUL
+  let accordionColor;
+
+  // Switch statement which looks for the resupply period  to match:
+  // soon (7), kind of soon (14), and not soon (30).
+  switch (resupplyPeriod) {
+    case 7:
+      accordionColor = 'yellow';
+      break;
+
+    case 14:
+      accordionColor = 'red';
+      break;
+
+    default:
+      accordionColor = 'green';
+  }
```

Comment:
This would be a nice place to implement constants for Soon(7), Kind of soon(14), and not so soon(30). That way you can implement the number app wide, but have the flexibility to change it later.

Reply:
Oh yeah that definitely makes sense, updating now.